lib/ActionGenerator.js

```diff
@@ -148,6 +151,17 @@ class ActionGenerator extends Generator {
     return testDestPath
   }
 
+  _writeActionE2ETest (actionName, tplActionTest, actionDestPath, tplContext = {}) {
+    // test/actions/actionName.test.js
+    const testDestPath = this.destinationPath('e2e', actionsDirname, actionName + '.e2etest.js')
+    // enriches tplContext with actionRelPath to be required from test file
+    tplContext = { actionRelPath: path.relative(path.dirname(testDestPath), actionDestPath), ...tplContext }
```

Comment:
here you don't need actionRelPath in the template context, actionName will already be part of tplContext